### ISSUE TITLE
sitewide styling issue: insufficient contrast

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#856]
Contrast between foreground and background colors must meet WCAG 2 AA contrast ratio thresholds. WCAG success criterion: color contrast of at least 4.5:1 for small text or 3:1 for large text, even if text is part of an image. Large text has been defined in the requirements as 18pt (24 CSS pixels) or 14pt bold (19 CSS pixels).  

See[ Deque Color Contrast Analyzer](https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=AxeChrome) for color hex code checker.  

Testing found insufficient contrast in the following places: 
- Text in the navigation bar at the top of virtually all pages of the site.
- In the hyperlinked blue text of the site footer.    
- In collection-level, catalog, and found-set views, item titles, and “Is part of” statements (blue text). 
- For logged-in users, breadcrumb text e.g. Home > Dashboard blue text.
- In Item views, all hyperlinked and facets metadata values, rendered in blue text.

**Comment by noraegloff** [comment]
Deque aXe tools severity level = serious
